src/pipelines/epidemiology/config.yaml

```diff
@@ -375,6 +375,11 @@ sources:
     fetch:
       - url: "https://raw.githubusercontent.com/nytimes/covid-19-data/master/us-counties.csv"
 
+  # Data sources for US_NY level 3 (NYC only)
+  - name: pipelines.epidemiology.us_ny_authority.NYCHealthDataSource
+    fetch:
+      - url: "https://raw.githubusercontent.com/nychealth/coronavirus-data/master/boro/boroughs-case-hosp-death.csv"
```

Comment:
I'm seeing hospitalizations sometimes greater than cases in this source, especially early on.  Not a blocker or something we need to fix, but something we might want to try to understand and caveat, especially for modelers who may want to compute hospitalization rates.

Reply:
Never mind, just realized this source shows new as opposed to total stats.